pb/contacts.proto

```diff
@@ -0,0 +1,14 @@
+syntax = "proto3";
+package pb;
+
+// Specifies how to parse contacts
+enum ContactType {
+    PHONE = 0;   // default contact type is a phone number (must contain country code)
+    EMAIL = 1;   // contact/connect via email address
+}
+
+// Contains the raw contact information along with parsing hints
+message Contact {
+    ContactType type = 1;  // determines how to parse the type of contact
+    string contact = 2;    // the string representation of the contact info
+}
```

Comment:
I wasn't sure if I should call this `Address` or `Contact` -- but loosely it's a human-centric method of identifying someone you want to contact. I envision that you'll pass a list of these to the server so that anyone can message you via a variety of mechanisms, e.g. either your email or phone number. These will be mapped to `boxid` which will be the primary addressable space in the platform.

Reply:
I agree that `Contact` is a bit more sensical than `Address`, since it doesn't have a physical connotation